Firestore/core/src/firebase/firestore/remote/grpc_nanopb_conversions.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2019 Google
```

Comment:
1. I don't see a particularly strong reason to split reader and writer into their own files, what do you think?
2. Feel free to suggest a different name for this file.

Reply:
I went with `grpc_nanopb.h`. My reasoning is that "util" is somewhat vague and is better used as a fallback name.